lib/commands.js

```diff
@@ -241,14 +241,15 @@ module.exports.commandContext = function(config, suffix, operations, callback) {
     templateBucket: config.templateBucket,
     overrides: {},
     oldParameters: {},
+    diffs: {},
```

Comment:
Maybe `approvedDiffs` or `preConfirmedDiffs` to make the intention clearer here.

Reply:
I'm going to stick with the language here -- this object is the collection of diffs that were performed as part of the current action, that is, the ones that have been approved by the current action. Then the context object simply conveys the diffs that were performed as the 3rd arg to the callback.

The parameter name in the overrides object could maybe be clarified -- `overrides.diffs` --> `overrides.confirmed` or `overrides.confirmedDiffs`. What do you think?

Reply:
Oh yea, I think that's the `diffs` I meant to refer to anyway - the param in overrides. 🤔 Thanks for changing it!